AsyncDisplayKit/ASCollectionView.h

```diff
@@ -420,6 +423,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)collectionViewUnlockDataSource:(ASCollectionView *)collectionView ASDISPLAYNODE_DEPRECATED;
 
+- (_Nonnull id<ASSectionContext>)collectionView:(ASCollectionView *)collectionView contextForSectionAtIndex:(NSInteger)sectionIndex;
```

Comment:
Exactly. This method is optional and if users implement it, they need to provide a context. 

That's said, in some cases, they may want to provide contexts for just a subset of sections. @Adlai-Holler Should we support it?